docs/alma/alma.rst

```diff
@@ -271,86 +284,83 @@ their types.
-   >>> print(uids)
-        Member ous id
-    -----------------------
-     uid://A002/X3216af/X31
-    uid://A002/X5a9a13/X689
-
-New to most recent versions of the library is that data does not need to be
-staged any longer. The ```stage_data``` method has been deprecated, but the
-new ```get_data_info``` method can be used instead to get information about
-the data such as the files, their urls, sizes etc:
+.. doctest-remote-data::
+    >>> import numpy as np
+    >>> from astroquery.alma import Alma
+    >>> m83_data = Alma.query_object('M83')
+    >>> uids = np.unique(m83_data['member_ous_uid'])
+    >>> print(uids) # doctest: +IGNORE_OUTPUT
+         member_ous_uid
+     -----------------------
+     uid://A001/X11f/X30
+     uid://A001/X122/Xf3
+
+The new ```get_data_info``` method can be used to get information about the
+data such as the file names, their urls, sizes etc (this method replaces
+```stage_data```, which served the same role in older versions of astroquery
+but is now deprecated):
 
 .. code-block:: python
-
+.. doctest-remote-data::
    >>> link_list = Alma.get_data_info(uids)
-   >>> link_list['content_length'].sum()
-   538298369462
-   >>> len(link_list)
-   >>> 47
 
 By default, ALMA data is delivered as tarball files. However, the content of
 some of these files can be listed and accessed individually. To get information
 on the individual files:
 
 .. code-block:: python
-
+.. doctest-remote-data::
     >>> link_list = Alma.get_data_info(uids, expand_tarfiles=True)
-    >>> len(link_list)
-    >>> 50
 
-You can then go on to download that data.  The download will be cached so that repeat
-queries of the same file will not re-download the data.  The default cache
-directory is ``~/.astropy/cache/astroquery/Alma/``, but this can be changed by
-changing the ``cache_location`` variable:
+You can then go on to download those files.  The download will be cached so
+that repeat queries of the same file will not re-download the data.  The
+default cache directory is ``~/.astropy/cache/astroquery/Alma/``, but this can
+be changed by changing the ``cache_location`` variable:
 
 .. code-block:: python
-
-   >>> myAlma = Alma()
-   >>> myAlma.cache_location = '/big/external/drive/'
-   >>> myAlma.download_files(link_list, cache=True)
+.. doctest-remote-data::
+   >>> myAlma = Alma() # doctest: +SKIP
+   >>> myAlma.cache_location = '/big/external/drive/' # doctest: +SKIP
+   >>> myAlma.download_files(link_list, cache=True) # doctest: +SKIP
 
 You can also do the downloading all in one step:
 
 .. code-block:: python
-
-   >>> myAlma.retrieve_data_from_uid(uids[0])
+.. doctest-remote-data::
+   >>> myAlma.retrieve_data_from_uid(uids[0]) # doctest: +SKIP
 
 Downloading FITS data
 =====================
 
-If you want just the QA2-produced FITS files, you can download the tarball,
-extract the FITS file, then delete the tarball:
+If you want just the QA2-produced FITS files, you can directly access the FITS
+files:
```

Comment:
the "download a tarball, extract, delete" approach was an old hack; this is the _right_ way to do this.